server/utils/notifications/google_chat_notification_service.py

```
@@ -0,0 +1,278 @@
+
+        # Try to update existing message, fallback to new message
+        if google_chat_message_name:
+            try:
+                result = client.update_message(
+                    message_name=google_chat_message_name,
+                    text=fallback_text,
+                    cards_v2=cards_v2,
+                )
+                if result:
+                    return True
+            except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)